FunctionalPreprocessing/FunctionalPreprocessing.XNAT_MARK_RUNNING_STATUS

```diff
@@ -167,17 +220,43 @@ main()
 
 	get_options "$@"
 
-	local directory=${XNAT_PBS_JOBS_RUNNING_STATUS_DIR}/${g_project}
-	local file="FunctionalPreprocessing.${g_subject}_${g_classifier}_${g_scan}.RUNNING"
+	local directory="${XNAT_PBS_JOBS_BUILD_DIR}/${g_project}/FunctionalPreprocessing.${g_session}_${g_scan}_RUNNING_STATUS"
+	local file="FunctionalPreprocessing.${g_session}_${g_scan}.RUNNING"
+	
+	local existing_file="${XNAT_PBS_JOBS_ARCHIVE_ROOT}/${g_project}/arc001/${g_session}/RESOURCES/RunningStatus/${file}"	
 	local path=${directory}/${file}
-
+	local g_dir=${directory}
```

Comment:
Don't name this variable `g_dir`. I'm sorry if this isn't as clear as it should be. Perhaps I should have documented it better somewhere. But a prefix of `g_` on a variable name is intended to indicate that the variable is global to the script. (All variables are global in bash scripts unless they are in a function and declared with the `local` keyword.)

So you created a `local` variable, but named it to indicate that it is a global variable.

Maybe you can just go ahead and use the `directory` variable that is already set above, and not even bother to create a `g_dir` variable?

Reply:
removed g_dir, it was not needed